lib/nerves_hub_web/channels/device_channel.ex

```diff
@@ -545,4 +560,9 @@ defmodule NervesHubWeb.DeviceChannel do
     |> assign(:device, device)
     |> assign(:deployment_channel, deployment_channel)
   end
+
+  defp device_deployment_change_jitter_seconds() do
```

Comment:
```suggestion
  defp device_deployment_change_jitter_ms() do
```
Changed to `_ms` since this returns milliseconds rather than seconds.